config.go

```diff
@@ -62,7 +62,7 @@ func LogOutputOption(args []string) (log.Output, error) {
 		case "syslog":
 			syslogOut, err := log.SyslogOutput()
 			if err != nil {
-				return nil, fmt.Errorf("failed to connect to syslog daemon: %v", err)
+				return nil, fmt.Errorf("failed to connect to syslog daemon: %w", err)
```

Comment:
Hmm that's a valid point. From my understanding `%w` will output the unwrapped error as a whole. Which is nice if some error is coming up and you can exactly see which error is the "root" cause. And thus I personally am using that. However if it breaks the abstraction I will happily revert it.

Reply:
"full error chain" is already visible in error messages formatted using %v, %w makes the chain accessible to the code and this means you are commiting to returning same wrapped errors in the future - the paragraph i linked from go blog shows how it may make it harder to change code later if used carelessly.